appinventor/appengine/src/com/google/appinventor/client/utils/html5dnd.js

```diff
@@ -82,6 +84,10 @@ function handleDroppedItem(item, cb) {
 }
 
 function importProject(droppedItem) {
+  if (typeof droppedItem == "string") {
+    droppedItem = {"name": droppedItem} // stop gap for handling different sources
+  }
+  console.log(droppedItem);
```

Comment:
Decide whether this still needs to stay.

Reply:
the `importProject` fxn is also called from 
`function checkValidDrop(e) {`
as well as from Ode.js

the dropped file is an key:value pair vs the repo url param is just a string.. So we can keep it as is or otherwise we would want to create a new object var rather than use `templatePath`

Reply:
I mean specifically the call to `console.log`. Yes, I agree the construction of droppedItem is necessary.